.github/workflows/build_and_test.yml

```diff
@@ -128,7 +128,7 @@ jobs:
       fail-fast: false
       matrix:
         suite: [integration, plugins]
-        split: [first (01-04), second (>= 05)]
+        split: [first, second]
```

Comment:
How can I tell which part of the split a test suite falls under if I just added a new one. Will I have to dig each run one by one?

Reply:
I believe this question refers to the randomization feature.  I'm not sure I understand - What specific reason would you have to want to know into which split your test falls?

Reply:
I see the point: If I add/modify a new test, I would like to check how long the test takes to run, and I will look for it in a green split.